apps/settings/test/marionette/app/regions/screen_lock.js

```diff
@@ -69,7 +69,8 @@ ScreenLockPanel.prototype = {
   },
 
   getPasscode: function() {
-    return this.client.settings.get('lockscreen.passcode-lock.code');
```

Comment:
If you don't want to anyone use this function, you should change the caller, not the callee and throwing an exception.